Cargo.lock

```diff
@@ -0,0 +1,4 @@
+[root]
```

Comment:
We shouldn't check in this file.

http://doc.crates.io/guide.html#cargotoml-vs-cargolock

> If you’re building a library that other projects will depend on, put Cargo.lock in your .gitignore. If you’re building an executable like a command-line tool or an application, check Cargo.lock into git. If you're curious about why that is, see "Why do binaries have Cargo.lock in version control, but not libraries?" in the FAQ.